portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -36,12 +38,12 @@
   @Override
   public void doGet(HttpServletRequest request, HttpServletResponse response) throws IOException {
     Query query = new Query("Comments").addSort("timestamp", SortDirection.DESCENDING);
+    query.addSort("username");
```

Comment:
Can you chain the `.addSort` calls together? Meaning `new Query("Comments").addSort("timestamp", SortDirection.DESCENDING).addSort("username");`.